cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/ExportXmlCompare.java

```diff
@@ -465,7 +465,7 @@ static ComparisonResult ignoringControlAreaNetInterchange(Comparison comparison,
             }
             if (comparison.getType() == ComparisonType.ATTR_NAME_LOOKUP) {
                 Comparison.Detail control = comparison.getControlDetails();
-                if (control.getValue().toString().equals("netInterchange")
+                if (control.getValue() != null && control.getValue().toString().equals("netInterchange")
```

Comment:
That part could be removed when the issue of the CGMES export altering the IIDM network will be tackled.